app/dawn-hub/Views/OnboardingViews/Step4_ChoosePlanView/ChoosePlanView.jsx

```diff
@@ -274,20 +208,88 @@ class ChoosePlanView extends React.Component {
+		const { actions, history } = this.props;
+		const { setSetupStep } = actions;
+		const cardClassNames = ClassNames('ChoosePlanView__card premium', {
+			checked
+		});
+		return (
+			<Fragment>
+				<div className="ChoosePlanView__cardOuter">
+					<div className={cardClassNames} onClick={handleClick} data-equalizer-watch>
+						<div className="ChoosePlanView__radioButtonContainer">
+							<RadioButton checked={checked} handleClick={handleClick} altDesign />
+						</div>
+						<div className="ghostery-premium-image-container">
+							<div className="ghostery-premium-image card-image-top" title="Ghostery Premium" alt="Ghostery Premium" />
+						</div>
+						<div className="ghostery-premium-image-background" />
+						<h2>Ghostery Premium</h2>
+						<div className="ChoosePlanView__price">
+							<Fragment>
+								<p className="ChoosePlanView__price-purple font-size-36">$11.99</p>
+								<p className="ChoosePlanView__price-purple sub-text font-size-12">{t('per_month')}</p>
+							</Fragment>
+						</div>
+						<p className="card-sub-header"><strong>{t('hub_upgrade_maximum_protection')}</strong></p>
+						<div className="ChoosePlanView__valuePropList">
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_private_search')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_tracker_protection')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_speedy_page_loads')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_intelligence_technology')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_ad_free')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_supports_ghosterys_mission')}
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								VPN
+							</div>
+							<div className="ChoosePlanView__cardSubCopy">
+								<span className="check blue" />
+								{t('ghostery_dawn_onboarding_unlimited_bandwidth')}
+							</div>
+						</div>
+					</div>
+				</div>
+				{showCTAButton && (
+					<a className="ChoosePlanView__premiumCTAButton" href={premiumCheckoutLink} target="_blank" rel="noreferrer" onClick={() => { setSetupStep({ setup_step: CHOOSE_PLAN, setup_number: 6, origin: ONBOARDING }); history.push('/onboarding/5'); }}>{t('ghostery_dawn_onboarding_upgrade')}</a>
+				)}
+			</Fragment>
+		);
+	};
+
 	render() {
 		const {
 			actions,
 			defaultSearch,
 			loggedIn,
-			next,
+			history,
```

Comment:
RIP `StepNavigator` ....you were so beautiful

Don't worry about removing it, I'll rip it out while you address these comments

Reply:
Sorry! I tried using it, but it brought me to a weird combination of a create account view and the Choose plan page